hyperspy/tests/drawing/test_plot_widgets.py

```diff
@@ -530,3 +531,20 @@ def count_calls(obj):
         assert count_calls.counter == 0
         assert im.axes_manager.navigation_axes[0].index == 0
         # drag down and check that it doesn't change
+
+
+def test_widgets_non_uniform_axis():
```

Comment:
Should there be a test that handles the mixed case of one axis uniform and another non-uniform?

Reply:
Good point, I had a look at this and actually it is more complicated than expected because there is no 2D ROI that word on non-uniform axes... so it is not possible to test a mix case (uniform and non-uniform). Fixing 2D ROI to work on non-uniform is a big rabbit hole from what I could see!

Reply:
I see, I reworded to "If None, it will be disabled (set to  ``False``) for signals containing non-uniform axes." to clarify.